packages/core/src/command/hl7-sftp-ingestion/sftp-ingestion-client.ts

```diff
@@ -0,0 +1,95 @@
+import { MetriportError } from "@metriport/shared";
+import { getSecretValueOrFail } from "../../external/aws/secret-manager";
+import { SftpClient } from "../../external/sftp/client";
+import { SftpConfig } from "../../external/sftp/types";
+import { Config } from "../../util/config";
+import {
+  decryptGpgBinaryWithPrivateKey,
+  getLahiePrivateKeyAndPassphrase,
+} from "./hl7-sftp-ingestion-direct";
+
+export type ReplicaConfig = { type: "local"; path: string } | { type: "s3"; bucketName: string };
+
+export class SftpIngestionClient extends SftpClient {
+  private readonly overridenLog: typeof console.log;
+
+  private constructor(sftpConfig: SftpConfig, overridenLog: typeof console.log) {
+    super(sftpConfig);
+    this.overridenLog = overridenLog;
+
+    const region = Config.getAWSRegion();
+    const bucketName = Config.getLahieIngestionBucket();
+
+    this.initializeS3Replica({ bucketName, region });
+  }
+
+  static async create(
+    overridenLog: typeof console.log,
```

Comment:
think you maybe resolved this previous comment but didnt fix it, just call this log instead of overriddenLog